### PR TITLE
Spy cx 4

### PR DESCRIPTION
**Pull request by mbarekh**
Support private send event

### CODE REVIEW

File: src/components/TransitionProperties.tsx [hunk 1]
```diff
@@ -106,18 +112,27 @@ const TransitionProperties = ({
...
-    return <Layer onClose={hideTransitionProperties} closer={true} align="right">
+    return <Layer onClose={hideTransitionProperties} align="right">
         <Form compact={false}>
-            <Header>
-                <Title>Send event form </Title>
-            </Header>
+            <Box direction="row" align="center" justify="center">
+                <Box align="start" justify="center" size="large" basis="1/2">
+                    <Title>
+                        Send Event
```
Comment (nicolaserny) [inline src/components/TransitionProperties.tsx]: Do not write labels directly in the code. Use a i18n resource file

File: src/core.ts [hunk 2]
```diff
@@ -28,8 +28,8 @@ export const subscribeAllStateMachines = (dispatch: Dispatch<XCSpyState>, compon
 
 export const snapshotEntryPoint = (dispatch: Dispatch<XCSpyState>, component: string, entryPoint: string): void => {
     sessionXCSpy.PromiseCreateSession
-        .then((session) => {
-            session.createSubscriber().getSnapshot(component, entryPoint, (items) => {
+        .then((session: Session) => {
+            session.createSubscriber().getSnapshot(component, entryPoint, (items: xcMessages.Packet[]) => {
```
Comment (nicolaserny) [inline src/core.ts]: What do you mean by Packet ?
Is it the right term ?

File: src/components/TransitionProperties.tsx (hunk 1 above)
Comment (mbarekh) [inline src/components/TransitionProperties.tsx]: done

File: src/core.ts (hunk 2 above)
Comment (mbarekh) [inline src/core.ts]: Packet={jsonMessage, stateMachineRef}. This object has been defined in the API.

File: src/locales/localeConfiguration.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { addLocaleData } from "react-intl";
+import * as en from "react-intl/locale-data/en";
+import localeData from "./data";
+
+export const getLocalizedResources = () => {
+    addLocaleData([...en]);
```
Comment (nicolaserny) [inline src/locales/localeConfiguration.ts]: You forgot to add fr support (your json resource already contains french translations)
const language = (navigator.languages && navigator.languages[0]) ||
                  navigator.language ||
                  navigator.userLanguage;

const languageWithoutRegionCode = language.toLowerCase().split(/[_-]+/)[0];

const messages = localeData[languageWithoutRegionCode] || localeData[language] || localeData.en;

this is the idea...

Comment (mbarekh) [inline src/locales/localeConfiguration.ts]: done

**Pull request merged**